blueprint/src/loops.tex

```diff
@@ -114,37 +117,45 @@ \section{Preliminaries}
   where we use $k_{i_0} = 0$ in the first equality.
 \end{proof}
 
-\begin{lemma}
-\label{lem:refined_caratheodory}
-  If a point $x$ of $E$ lies in the convex hull of a set $P$, then $x$
-  can be written as the convex combination of at most $d + 1$ affinely
-  independent points in $P$ with positive coefficients.
-\end{lemma}
-
-\begin{proof}
-  \uses{lem:caratheodory}
-  \cref{lem:caratheodory} gives points $p_i$ in $P$ and non-negative
-  weights $w_i$ such that $x = \sum w_i p_i$ for $0 ≤ i ≤ k$ where
-  $k ≤ d$. We first discard every point with vanishing weights.
-  Now suppose the $p_i$'s are not affinely independent. Then there is
-  some relation $p_{i_0} = \sum λ_j p_j$ with positive weights
-  $λ_j$. We can then discard $p_{i_0}$ and replace each remaining
-  $w_j$ by $w_j + w_{i_0}λ _j$.
-\end{proof}
-
 
 \begin{definition}
   \label{def:surrounds_points}
   \lean{surrounding_pts}
   \leanok
-  Let $F$ be a real vector space with dimension $d$. A point $x$ in $E$
```

Comment:
I'm sorry about changing my mind about the name. In the applications all surrounding happens in the target space that is called F. But I'm not consistent, even in two successive sentences.

Reply:
No problem, I think we're locally consistent now. Perhaps later we'll bring some global consistency.